docs/guide/user_management/user_management.md

```diff
@@ -17,6 +17,41 @@ You can [customize it according to your needs](#customize-login-form).
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
+### Add forget password option
+
+You can add a "forgot password" option to the site front to allow users of a specific site request a password change.
+
+Edit `config/packages/ezplatform.yaml`. The templates are defined with the following configuration:
+
+```yaml
+ezplatform:
+    system:
+        <siteaccess>:
+            user_forgot_password:
+                templates:
+                    form: <path_to_template>
+                    mail: <path_to_template>
+```
+
+The [default templates](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/views/themes/admin/account/forgot_password/) for forgot password form and email are located in `/src/bundle/Resources/views/themes/admin/account/` in `ezsystems/ezplatform_admin_ui`.
```

Comment:
To be precise the defaults one are located in https://github.com/ezsystems/ezplatform-user/tree/master/src/bundle/Resources/views, those in adminUI are specific one to our adminUI (I would mention both in docs).